Library/Formula/libxslt.rb

```diff
@@ -31,3 +40,31 @@ def caveats; <<-EOS.undent
     EOS
   end
 end
+__END__
```

Comment:
Can we submit a patch upstream for this? Would rather it's accepted upstream before we merge this, thanks!